R/chart.R

```diff
@@ -0,0 +1,87 @@
+#' Chart the application of a function along coordinates
+#'
+#' \code{chart()} modifies a function that takes a list to make it act
```

Comment:
yes that's not well written and maybe the whole idea of a factory function is not necessary.

Was I clear on how `map_along()` works at least?